### DISCUSSION TITLE
Where is ddd helper?

### DISCUSSION THREAD

**Discussion by sajt** [spatie/laravel-ignition#80] [General]
This is the question. Where is ddd helper method, what was in facade/ignition.

**Comment by freekmurze** [comment]
We removed it, and might bring it back in the future.

**Comment by AlexVanderbist** [comment]
Hey @sajt, feel free to add it to your own helper file (or PR it to this package if you feel like it). Here's the source code that works with spatie/laravel-ignition

```php
if (! function_exists('ddd')) {
    function ddd()
    {
        $args = func_get_args();

        if (count($args) === 0) {
            throw new Exception('You should pass at least 1 argument to `ddd`');
        }

        call_user_func_array('dump', $args);

        $renderer = app()->make(\Spatie\LaravelIgnition\Renderers\ErrorPageRenderer::class);

        $exception = new Exception('Dump, Die, Debug');

        $renderer->render($exception);

        die();
    }
}
```

**Comment by freekmurze** [comment]
`ddd` is available again in the latest version of the package 👍

**Comment by sajt** [comment]
And how does it work? If I use it it is only generate an exception.
